libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/TabsView/TabsView.tsx

```diff
@@ -164,7 +164,7 @@ export class TabsView extends React.PureComponent<
                 this.props.dataset.class_names && (
                   <>
                     <div className={classNames.sectionHeader}>
-                      <Text variant={"xxLarge"}>
+                      <Text as="h2" variant={"xxLarge"}>
```

Comment:
why not just use <h2>?

Reply:
in that case we need to adjust the styles a little bit. This is a simpler fix, same as  ([#2415](vscode-file://vscode-app/c:/Users/tongy/AppData/Local/Programs/Microsoft%20VS%20Code/resources/app/out/vs/code/electron-sandbox/workbench/workbench.html))